File: Array-List/Python/basic.py
"""
?Create a print 
arr = [10,20,30,40,50]
print(arr)
"""

"""
?Input of Arrays

# other way
#  n = int(input("Enter size of array: ")) # Read size
# arr = list(map(int, input("Enter elements: ").split())) # Read multiple inputs

# print("Array elements:", arr)
"""

n = int(input("Enter size of array: "))
arr = []

for _ in range(n):
     arr.append(int(input("Enter the numbers: ")))

print(arr)
print(len(arr))

File: Array-List/JavaScript/basic.js
/*
?Create a print 
let arr = [10,20,30,40,50];
console.log(arr);
*/

/*
?Input in arrays
*/
let n = parseInt(prompt("Enter the size of arrays: "));
let arr = [];


for(let i = 0; i < n; i++){
    arr.push(parseInt(prompt(`Enter the numbers: ${i+1}`)));
}

console.log(arr);
console.log(arr.length)